.babelrc

```diff
@@ -10,12 +9,6 @@
             }
         ],
         "@babel/preset-react",
-        [
-            "react-app",
-            {
-                "absoluteRuntime": false
-            }
-        ]
```

Comment:
react-app caused error w/ transpile preventing installation of code within `dist` directory

Reply:
this file is the same as `SOF-5238`, but it is not showing them as the same.